### PR TITLE
enhance output instuction when creating cc cluster with legacy config

### PR DESCRIPTION
**Pull request by hongtaiwang**
### What this PR does / why we need it

When creating cluster using cluster yaml, the validation does not read configurations from the yaml context. So if none custom flags are explicitly set in this step, tanzu cli will validate with defualt value

ALL customized flags are required to set when creating cluster with a cc cluster yaml.

### Which issue(s) this PR fixes

Fixes #

### Release note

```release-note

```

### CODE REVIEW

File: tkg/client/cluster.go [hunk 1]
```diff
@@ -180,7 +180,7 @@ func (c *TkgClient) CreateCluster(options *CreateClusterOptions, waitForCluster
 			// If `features.cluster.auto-apply-generated-clusterclass-based-configuration` feature-flag is not activated
 			// log command to use to create cluster using ClusterClass based config file and return
 			if !c.IsFeatureActivated(constants.FeatureFlagAutoApplyGeneratedClusterClassBasedConfiguration) {
-				log.Warningf("\nTo create a cluster with it, use")
+				log.Warningf("\nTo create a cluster with it, please use the following command together with customize flags like --tkr, -v, etc.")
```
Comment (wjun) [inline tkg/client/cluster.go:183]: Please provide more context why these customize flags are required.

Comment (hongtaiwang) [reply tkg/client/cluster.go:183]: Indeed, all customized flags are required.
When creating cluster using cluster yaml, the validation does not read configurations from the yaml context. So if none custom flags are explicitly set in this step, tanzu cli will validate with defualt value

Comment (hongtaiwang) [reply tkg/client/cluster.go]: updated the wording to make it more clear

File: tkg/client/cluster.go [hunk 2]
```diff
@@ -180,7 +180,7 @@ func (c *TkgClient) CreateCluster(options *CreateClusterOptions, waitForCluster
 			// If `features.cluster.auto-apply-generated-clusterclass-based-configuration` feature-flag is not activated
 			// log command to use to create cluster using ClusterClass based config file and return
 			if !c.IsFeatureActivated(constants.FeatureFlagAutoApplyGeneratedClusterClassBasedConfiguration) {
-				log.Warningf("\nTo create a cluster with it, use")
+				log.Warningf("\nTo create a cluster with it, please use the following command together with all customized flags set in the last step. e.g. --tkr, -v")
```
Comment (xiujuanx) [inline tkg/client/cluster.go:183]: @hongtaiwang may I know why the feature gate `auto-apply-generated-clusterclass-based-configuration` must to to be enabled when create a cluster with a kubernetes-style yaml file? 
As far as I know, when create a classy workload-cluster with k8s-style yaml file, code logic will not go here. It should be go into here: https://github.com/vmware-tanzu/tanzu-framework/blob/main/tkg/client/cluster.go#L147-L156. So enable this feature gate with a k8s-style yaml file may not make sense.

Also the error is reported by the user is more like a mapping error for `VSPHERE_TEMPLATE` var. At the first step, user specifies `--tkr v1.23.16---vmware.1-tkg.2`, but why the value of this var is parsed to be `/TSV_DC/vm/ubuntu_temp_v1.23.16`? What's the correct template name for v1.23.16---vmware.1-tkg.2 tkr?

BTW, the `-v` is used for debugging AFAIK, so may it's not necessary. Or does it do something else?

Comment (hongtaiwang) [reply tkg/client/cluster.go:183]: better to have a zoom call to carify this:)

**Review by wjun — APPROVED** [review]
LGTM.

**Pull request merged**